src/shared/components/flexbox/FlexBox.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+	Copyright (c) 2015 Weave Visual Analytics, Inc.
+
+	This Source Code Form is subject to the terms of the
+	Mozilla Public License, v. 2.0. If a copy of the MPL
+	was not distributed with this file, You can obtain
+	one at https://mozilla.org/MPL/2.0/.
+*/
+
+import * as React from "react";
+import * as _ from 'lodash';
+import styles from './styles.module.scss';
+import classNames from "shared/lib/classNames";
+
+export interface IFlexBoxProps <T> extends React.HTMLProps<T>
+{
+	flex?:number;
+	padded?:boolean;
+	overflow?:boolean;
+}
+
+export class IFlexBoxProps<T>
```

Comment:
I guess I don't see the point, it seems sort of confusing. But if you say it makes something cleaner then I believe it.